src/main/java/de/ellpeck/rockbottom/item/ItemBowl.java

```diff
@@ -17,6 +17,6 @@ public boolean allowsLiquid(ItemInstance instance, ResourceName name) {
 
     @Override
     public int getCapacity(ItemInstance instance) {
-        return 5;
+        return 6;
```

Comment:
Why this change?

Reply:
So that bowl take half a tiles worth of liquid as initially intended

Reply:
ok